pkgs/by-name/re/restique/package.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  libsForQt5,
+  fetchgit,
+  cmake,
+  libsecret,
+  restic,
+}:
+
+libsForQt5.mkDerivation {
```

Comment:
According to https://github.com/NixOS/nixpkgs/issues/180841, it might be better to use `stdenv.mkDerivation`.

Reply:
I'm not sure on what to do here